.github/workflows/test-e2e-cli.yml

```diff
@@ -13,33 +13,33 @@ env:
   TEST_PARALLELISM_PKG: 1
   TEST_PROJECTS_LINUX: |
     [
-      {"host":"connection.keboola.com","project":8764,"stagingStorage":"s3","token":"${{ secrets.TEST_KBC_PROJECT_8764_TOKEN }}"},
-      {"host":"connection.keboola.com","project":8765,"stagingStorage":"s3","token":"${{ secrets.TEST_KBC_PROJECT_8765_TOKEN }}"},
-      {"host":"connection.keboola.com","project":8766,"stagingStorage":"s3","token":"${{ secrets.TEST_KBC_PROJECT_8766_TOKEN }}"},
-      {"host":"connection.north-europe.azure.keboola.com","project":10890,"stagingStorage":"abs","token":"${{ secrets.TEST_KBC_PROJECT_10890_TOKEN }}"},
-      {"host":"connection.north-europe.azure.keboola.com","project":10891,"stagingStorage":"abs","token":"${{ secrets.TEST_KBC_PROJECT_10891_TOKEN }}"},
-      {"host":"connection.north-europe.azure.keboola.com","project":10892,"stagingStorage":"abs","token":"${{ secrets.TEST_KBC_PROJECT_10892_TOKEN }}"}
+      {"host":"connection.keboola.com","backend":"snowflake","project":8764,"stagingStorage":"s3","token":"${{ secrets.TEST_KBC_PROJECT_8764_TOKEN }}"},
+      {"host":"connection.keboola.com","backend":"snowflake","project":8765,"stagingStorage":"s3","token":"${{ secrets.TEST_KBC_PROJECT_8765_TOKEN }}"},
+      {"host":"connection.keboola.com","backend":"snowflake","project":8766,"stagingStorage":"s3","token":"${{ secrets.TEST_KBC_PROJECT_8766_TOKEN }}"},
+      {"host":"connection.north-europe.azure.keboola.com","backend":"snowflake","project":10890,"stagingStorage":"abs","token":"${{ secrets.TEST_KBC_PROJECT_10890_TOKEN }}"},
```

Comment:
Please, add `backend` field after the `stagingStorage` field, to improve readability.